### PR TITLE
Rework tutorials

### PR DESCRIPTION
**Pull request by boriskaus**
Various fixes; also went over all tutorials and created Literate versions of the key ones.

### CODE REVIEW

File: docs/src/man/tutorial_GMT_Topography_GeologicalMap.md [hunk 1]
```diff
@@ -1,30 +1,23 @@
-# 8 - Extract ETOPO1 topographic data using GMT.jl and drape a geological map on top of the topography (given as raster graphics)
+# Extract topographic data uand drape a geological map on top of it (given as raster graphics)
```
Comment (aelligp) [inline docs/src/man/tutorial_GMT_Topography_GeologicalMap.md:1]: typo

File: docs/src/man/tutorial_loadregular3DSeismicData_netCDF.md [hunk 2]
```diff
@@ -104,15 +104,15 @@ julia> depth = -1 .* depth
 Note that we multiplied depth with -1. This is necessary to make depth to be negative, as that is what `GeophysicalModelGenerator.jl` expects.
 
 #### 3. Reformat the coordinate data
-In the netCDF file, coordinates are given as 1D vectors denoting the location of nodes in a regular grid. However, `GeophysicalModelGenerator.jl` expects true 3D data, where each data point is assigned a latitude,longitude, depth and the respective property (here: Vs). To generate this full regular 3D grid, do the following:
+In the netCDF file, coordinates are given as 1D vectors denoting the location of nodes in a regular grid. However, `GeophysicalModelGenerator.jl` expects true 3D data, where each data point is assigned a latitude,longitude, depth and the respective property (here: `Vs`). To generate this full regular 3D grid, do the following:
 ```julia-repl
```
Comment (aelligp) [inline docs/src/man/tutorial_loadregular3DSeismicData_netCDF.md:108]:
```suggestion
```julia
```

Comment (aelligp) [inline docs/src/man/tutorial_loadregular3DSeismicData_netCDF.md:108]:
```suggestion
```julia
```

File: src/LaMEM_io.jl
Comment (aelligp) [inline src/LaMEM_io.jl:1]: Can the Codecov tests be addressed here w/out bigger changes?

File: src/data_types.jl
Comment (aelligp) [inline src/data_types.jl:1]: Codecov

**Review by aelligp — CHANGES_REQUESTED** [review]
There are several lines not covered by tests. if feasible, can they be addressed?

File: src/LaMEM_io.jl
Comment (boriskaus) [reply src/LaMEM_io.jl:1]: I think it is more something for a different PR

File: docs/src/man/tutorial_GMT_Topography_GeologicalMap.md (hunk 1 above)
Comment (aelligp) [inline docs/src/man/tutorial_GMT_Topography_GeologicalMap.md:1]:
```suggestion
# Extract topographic data and drape a geological map on top of it (given as raster graphics)
```

**Review by aelligp — APPROVED** [review]

File: docs/src/man/tutorial_loadregular3DSeismicData_netCDF.md [hunk 3]
```diff
@@ -92,7 +92,7 @@ latitude                                        FLOAT                   latitude
 longitude                                       FLOAT                   longitude
 Vs                                              FLOAT                   longitude latitude depth
 ```
-Here we can see that there are four variables in this file, three of them (depth,latitude, longitude) having a single dimension and the fourth one (Vs) having dimensions of the three previous variables. The three one-dimensional vectors therefore denote a regular grid of coordinates defining the locations where Vs is stored.
+Here, we can see that there are four variables in this file, three of them (`depth`,`latitude`, `longitude`) having a single dimension and the fourth one (`Vs`) having dimensions of the three previous variables. The three one-dimensional vectors therefore denote a regular grid of coordinates defining the locations where `Vs` is stored.
 To load this data, we can now simply use the command *ncread*:
 ```julia-repl
```
Comment (aelligp) [inline docs/src/man/tutorial_loadregular3DSeismicData_netCDF.md:97]: is there a need to have Julia-repl here?

Comment (boriskaus) [reply docs/src/man/tutorial_loadregular3DSeismicData_netCDF.md:97]: I guess both work and make Julia like syntax highlighting

**Pull request merged**